website_crm_privacy_policy/templates/contactus.xml

```diff
@@ -9,9 +9,7 @@
             expr="//form//div/div/span[@id='s_website_form_result']/.."
             position="before"
         >
-            <div
-                class="form-group col-12 s_website_form_field s_website_form_required_custom"
-            >
+            <div class="form-group col-12">
                 <div class="form-check">
```

Comment:
Is all this stuff needed? Can this be replaced by enabling such check that you mention?

Reply:
After analysing it, if the option is enabled in the editor, there is no way to do it specifically only for this form, it would be enabled for all the forms that are inserted from the snippet, in addition we would add the dependency to this module. 
The other option is to recreate the behaviour of the website_form_require_legal module which involves defining new js code to add the terms acceptance checkbox in addition to the dependency on website_form_require_legal.
This modification has the least impact on this module and also does not imply incompatibility with website_form_require_legal as it will eventually be replaced by website_form_require_legal.